Airports/EGWU/SMR/Regions.txt

```diff
@@ -0,0 +1,562 @@
+; - Background
+REGIONNAME Northolt
+smrmilgrass N051.32.55.544 W000.24.39.300
+N051.32.56.268 W000.24.35.290
+N051.32.55.997 W000.24.30.957
+N051.32.55.324 W000.24.30.602
+N051.32.59.363 W000.24.12.422
+N051.33.01.170 W000.24.13.477
+N051.33.01.480 W000.24.12.200
+N051.33.12.525 W000.24.21.464
+N051.33.12.975 W000.24.19.979
+N051.33.14.875 W000.24.21.407
+N051.33.16.427 W000.24.16.827
+N051.33.18.344 W000.24.18.474
+N051.33.20.314 W000.24.20.515
+N051.33.22.676 W000.24.23.392
+N051.33.22.378 W000.24.23.955
+N051.33.22.571 W000.24.26.330
+N051.33.22.117 W000.24.34.903
+N051.33.24.057 W000.24.39.605
+N051.33.24.285 W000.24.44.142
+N051.33.26.518 W000.24.51.076
+N051.33.21.710 W000.25.13.520
+N051.33.25.429 W000.25.15.625
+N051.33.26.046 W000.25.17.791
+N051.33.25.887 W000.25.18.669
+N051.33.26.492 W000.25.24.495
+N051.33.29.101 W000.25.31.043
+N051.33.28.147 W000.25.38.995
+N051.33.23.230 W000.25.50.444
+N051.33.19.882 W000.25.52.147
+N051.33.16.300 W000.25.50.792
+N051.33.08.779 W000.25.43.246
+N051.33.05.417 W000.25.57.590
+N051.33.01.958 W000.26.00.502
+N051.32.58.922 W000.26.00.380
+N051.32.55.316 W000.25.10.461
+N051.32.53.551 W000.24.39.685
+; - D
```

Comment:
```suggestion

;D
```

Conventions are that a line break is generally added before each new region section

Reply:
I have updated this for all the pending SMR's